xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -652,7 +676,7 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
 
     CLog::Log(
         LOGINFO,
-        "CDVDVideoCodecAndroidMediaCodec::Open: Secure decoder requested: {} (stream flags: {})",
+        "CDVDVideoCodecAndroidMediaCodec::Open Secure decoder requested: {} (stream flags: {})",
```

Comment:
since you are cleaning logs, you can probably simplify everything with `LogF`
`CLog::LogF(LOGINFO,"Secure decoder requested: {} (stream flags: {})", ...`

Reply:
`LogF()` only shows the function, the class is missing, I think it is better to leave it as it is since it identifies the decoder